docs: Add repo.lock troubleshooting steps to readme (#2111)
ipfs/ipfs-desktop: README.md

You can change this in the IPFS Desktop config file by selecting `Open Configuration File` from the `Advanced` submenu.

> ### I got a repo.lock error. How do I resolve this?

Reviewer: ```suggestion
### I got a `repo.lock` error. How do I resolve this?
```

> In general, this means that a previous process was unable to remove the repo lock (indicator that file is in use) from the repo file. This is supposed to be handled automatically, but sometimes it isn't. If you get this error, you can generally safely delete this file after shutting down any running ipfs daemon's or applications. Simple process is as follows:

Reviewer: ```suggestion
In general, this means that a previous process was unable to remove the repository lock (indicator that file is in use) from the repository directory. This is supposed to be handled automatically, but sometimes it isn't. If you get this error, you can generally safely delete this file after shutting down any running IPFS daemon's or applications. Simple process is as follows:
```

1. Stop ipfs processes

> 2. Manually delete lock file

Reviewer: ```suggestion
2. Manually delete lock file, located within the [repository](#how-does-ipfs-desktop-select-the-ipfs-repo-location)
```

Author: ooh nice.

> 3. Attempt to start ipfs desktop (or other process that received the repo.lock error) again.

Reviewer: ```suggestion
3. Attempt to start ipfs desktop (or other process that received the `repo.lock` error) again.
```
